src/cli.c

```diff
@@ -28,153 +28,228 @@
 #include "tjs.h"
 #include "version.h"
 
+#include <stdlib.h>
 #include <string.h>
 
+#define EXIT_INVALID_ARG 2
 
-static int eval_buf(JSContext *ctx, const void *buf, int buf_len, const char *filename, int eval_flags) {
+#define OPT_PREFIX '-'
+#define OPT_ASSIGN '='
+
+#define is_longopt(opt, str) (opt.name && !strncmp(opt.name, str, opt.length))
+
+typedef struct CLIOption {
+    char key;
+    char *name;
+    unsigned length;
+} CLIOption;
+
+typedef struct Flags {
+    unsigned interactive, empty_run, strict_module_detection;
```

Comment:
Please put each field in a separate line. Also, we can use bool here.

Reply:
I wanted to have one per line but clang format keeps bringing them together because line does not exceed certain limit.

Reply:
Add a disable directive if need be